Source/WebKit/UIProcess/API/glib/WebKitUserContentManager.cpp

```diff
@@ -224,12 +256,158 @@ void webkit_user_content_manager_remove_all_scripts(WebKitUserContentManager* ma
     manager->priv->userContentController->removeAllUserScripts();
 }
 
+/**
+ * WebKitUserScriptMessageReply: (ref-func webkit_user_script_message_reply_ref) (unref-func webkit_user_script_message_reply_unref)
```

Comment:
WebKitUserScriptMessageReply -> WebKitScriptMessageReply

Reply:
Just would like to confirm that I should refactor all `WebKitUserScriptMessageReply` into `WebKitScriptMessageReply`, along with the naming of related methods, correct? (The rest of the suggestions have mostly been resolved, and thank you so much for all the feedback!)

Reply:
Yes, if you have addressed all other review comments

Reply:
WebKitUserScriptMessageReply could be confusing with something related to WebKitUserScript and the signal name is script-message-with-reply-received (no user-script-message-with-reply-received) so it's consistent with signal name too.